e2e-tests/security-context/compare/cronjob_each-hour-pvc-oc.yml

```diff
@@ -0,0 +1,69 @@
+apiVersion: batch/v1beta1
```

Comment:
this file is identical to `e2e-tests/security-context/compare/cronjob_each-hour-pvc.yml` so I think it is not needed